### PR TITLE
Initial support for santa logs #1556

### PR DESCRIPTION
**Pull request by aguilajesus**
Initial support for santa logs #1556

### CODE REVIEW

**Comment by joachimmetz** [comment]
Assigning to @Onager due to lack of available time

File: plaso/parsers/santa.py [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+    action (str): event type recorded by Santa.
+    mount (str): disk mount point.
+    volume (str): disk volume name.
+    bsd_name (str): disk BSD name.
+    fs (str): disk volume kind.
+    model (str): disk model.
+    serial (str): disk serial.
+    bus (str): device protocol.
+    dmg_path (str): DMG file path.
+    appearance (str): disk appearance date.
```
Comment (Onager) [inline plaso/parsers/santa.py]: This looks like a separate (and very useful!) timestamp. What do you think about adding this as a separate event as well?

Comment (aguilajesus) [reply plaso/parsers/santa.py]: I agree, but I am not 100% sure the field is consistent across all DISKAPPEAR events; if appearance field is present we can create another SantaMountEventData() event with definitions.TIME_DESCRIPTION_FIRST_CONNECTED.

**Comment by aguilajesus** [comment]
@Onager PTAL

**Pull request merged**